implants/Cargo.toml

```diff
@@ -38,7 +38,8 @@ rust-embed = "6.6.0"
 serde = "1.0"
 serde_json = "1.0.87"
 sha256 = "1.0.3"
-starlark = { git = "https://github.com/facebookexperimental/starlark-rust", rev = "acf638430a00ca3855862e8c669670e1adaa42aa" }
+starlark = { git = "https://github.com/facebookexperimental/starlark-rust", rev = "1109aaf9b700d7cdd7ba48986aa650221b70a22f" }
```

Comment:
i dream of a day we use releases instead of commit hashes.

Reply:
I'll play with it see if we can.
From what I've seen the way starlark rust vendors their dependencies doesn't guarantee a release of allocative will align with a release of starlark eg.
Starlark 0.9.0 might use allocative aaabc123 which is two commits ahead of allocative 0.0.2

Reply:
We can switch to 0.9.0 and allocative 0.3.0 those were released on the same day and seem to work together.
Theres some commits after 0.9.0 though that make it easier to work with starlark types so i'd like to keep with the commit for now.